src/hooks/useSelectionLogic.ts

```diff
@@ -219,3 +230,7 @@ export function useSelectionLogic<T extends HTMLElement>({
     cancelCurrentSelection,
   };
 }
+
+function isValidSelectionStartDefault(box: Box): boolean {
```

Comment:
Could we rename it to be more descriptive? Like `isMinumumBoxArea` or something like that?